src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
 export interface Figure {
-
+  color: 'red' | 'green' | 'blue';
+  shape: string;
+  radius?: number;
+  getArea: () => number;
```

Comment:
This interface should store only those properties, that should be in each figure

Reply:
It's better to create separate types to define what values can be in color and shape properties